requirements.txt

```
@@ -2,6 +2,8 @@
 ##### pip requirements ######
 #
 numpy<2
+pyzag
+nbdime
 -r doc/requirements.txt
 -r scripts/requirements.txt
 -r python/tests/requirements.txt
```

```suggestion
pyzag
-r doc/requirements.txt
-r scripts/requirements.txt
-r python/tests/requirements.txt
-r python/examples/requirements.txt
```

Then create `python/examples/requirements.txt` with
```
nbdime
pyro
tqdm
```

---

Good idea, and good catch on the other dependencies.